code/game/machinery/telecomms/machines/message_server.dm

```diff
@@ -179,11 +182,11 @@
 	return data["targets"][1]
 
 /datum/signal/subspace/messaging/pda/proc/format_message()
-	if (logged && data["photo"])
```

Comment:
What happened to photos?

Reply:
this version is currently not representative with the final product; photos will remain

Reply:
If you need to look at similar implementation of tgui photos, we have that now in newscasters.